src/server/scripts/Commands/cs_anticheat.cpp

```diff
@@ -85,11 +97,56 @@ class anticheat_commandscript : public CommandScript
         Player* pTarget = player->GetConnectedPlayer();
 
         // teleport both to jail.
-        WorldLocation jail {1, 16226.5f, 16403.6f, -64.5f, 3.2f};
-        pTarget->TeleportTo(jail);
-        handler->GetPlayer()->TeleportTo(jail);
-        pTarget->SetHomebind(jail, 876);
+        handler->GetSession()->GetPlayer()->TeleportTo(1, 16226.5f, 16403.6f, -64.5f, 3.2f);
+
+        WorldLocation loc;
+        loc = WorldLocation(1, 16226.5f, 16403.6f, -64.5f, 3.2f);// GM Jail Location
+        pTarget->TeleportTo(loc);
+        pTarget->SetHomebind(loc, 876);// GM Jail Homebind location
+        pTarget->CastSpell(pTarget, SHACKLES);// shackle him in place to ensure no exploit happens for jail break attempt
+        Aura* dungdesert = pTarget->AddAura(LFG_SPELL_DUNGEON_DESERTER, pTarget);// LFG_SPELL_DUNGEON_DESERTER
+        Aura* bgdesert = pTarget->AddAura(BG_SPELL_DESERTER, pTarget);// BG_SPELL_DESERTER
+        dungdesert->SetDuration(-1);
+        bgdesert->SetDuration(-1);
+
+        return true;
+    }
+
+    static bool HandleAnticheatParoleCommand(ChatHandler* handler, Optional<PlayerIdentifier> player)
+    {
+        if (!sWorld->getBoolConfig(CONFIG_ANTICHEAT_ENABLE))
+            return false;
+
+        if (!player)
+            player = PlayerIdentifier::FromTarget(handler);
+        if (!player || !player->IsConnected())
+        {
+            handler->SendSysMessage(LANG_PLAYER_NOT_FOUND);
+            handler->SetSentErrorMessage(true);
+            return false;
+        }
 
+        Player* pTarget = player->GetConnectedPlayer();
+
+        WorldLocation Aloc;
+        WorldLocation Hloc;
+        Aloc = WorldLocation(0, -8833.37f, 628.62f, 94.00f, 1.06f);// Stormwind
+        Hloc = WorldLocation(1, 1569.59f, -4397.63f, 16.06f, 0.54f);// Orgrimmar
```

Comment:
I wonder, can these 4 lines be written on 2 lines ?
Like `WorldLocation Aloc = WorldLocation(0, -8833.37f, 628.62f, 94.00f, 1.06f);// Stormwind`

Reply:
https://github.com/TrinityCore/TrinityCoreCustomChanges/pull/73/commits/7441be6078e8b03adf999d1c52a7be8dccd8b89d